index.html

```diff
@@ -17,10 +17,80 @@
 <body>
 
     <div class="container">
+
+        <div class="row justify-content-end">
+            <button type="button" class="btn btn-primary">Contact</button>
+            <button type="button" class="btn btn-primary">Refrences</button>
+            <button type="button" class="btn btn-primary">Skills</button>
+        </div>
+
       <div class="row">
           <div class="col-sm-12">
-              <h1>Start Here!</h1>
+              <ul class="nav nav-tabs">
+                  <li class="nav-item dropdown">
+                    <a class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" role="button" aria-haspopup="true" aria-expanded="false">Social Media</a>
+                    <div class="dropdown-menu">
+                      <a class="dropdown-item" href="#">Linkedin</a>
+                      <div class="dropdown-divider"></div>
+                      <a class="dropdown-item" href="#">Facebook</a>
+                      <a class="dropdown-item" href="#">Twitter</a>
+                      <a class="dropdown-item" href="#">Instagram</a> 
+                    </div>
+                  </li>
+                </ul>     
+          </div>
+          
+          <div class="row">
+              <div class="col-sm-12">
+                  <!-- some content here -->
+                  <div class="jumbotron jumbotron-fluid">
+                      <div class="container">
+                        <h1 class="display-4">Andrew Kolumbic's Resume</h1>
+                        <p class="lead">I will not beg for a job, but please, for the love of God, hire me this instant.</p>
+                      </div>
+                    </div>
+              </div>
           </div>
+
+          <table class="table table-sm">
+              <thead>
+                <tr>
+                  <th scope="col"> </th>
+                  <th scope="col">Company Name</th>
+                  <th scope="col">Position Held</th>
+                  <th scope="col">Dates</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <th scope="row"> </th>
+                  <td>Uber/Lyft</td>
+                  <td>Independent Contractor</td>
+                  <td>Oct 2015-Dec 2017</td>
+                </tr>
+                <tr>
+                  <th scope="row"> </th>
+                  <td>Kroger (Ralph's)</td>
+                  <td>General Merchandise Clerk</td>
+                  <td>Sept 2014-Dec 2015</td>
+                </tr>
+                <tr>
+                  <th scope="row"> </th>
+                  <td>Seigel Food Services LLC (Taco Bell Franchise)</td>
+                  <td>General Manager, Torrance</td>
+                  <td>Feb 2008-Sept 2014</td>
+                </tr>
+              </tbody>
+            </table>
+
+            <div class="card bg-light text-blue">
```

Comment:
I was going to ask you to please use alt text but there it is. Also this picture is probably the best picture ever.

Probably more that can be done with it but I'm pretty sure we touched on that yesterday already. :+1: